src/tag/tag-filter.component.ts

```diff
@@ -1,11 +1,12 @@
-import { Component, HostBinding } from "@angular/core";
+import { Component, Output, EventEmitter, HostBinding } from "@angular/core";
```

Comment:
Another small style change, but we try to [limit imports to 3 on one line](https://github.com/IBM/carbon-components-angular/wiki/Style-guide#import-ordering)

So:
```suggestion
import { 
	Component, 
	Output, 
	EventEmitter, 
	HostBinding 
} from "@angular/core";
```